### PR TITLE
Oct 10 creating initial file structure

### PR DESCRIPTION
**Pull request by Dmitry-Titov-K**

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
```
Comment (kogonidze) [inline src/app/App.tsx:2]: Empty space

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": false,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "jsx": "react",
+    "removeComments": true,
+
```
Comment (kogonidze) [inline tsconfig.json:21]: Empty space

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    },
+
+    target: ['web'],
+
+    module: {
+        rules: [{
+                test: /\.(ts|js)x?$/,
+                exclude: /node_modules/,
+                use: { loader: "babel-loader" },
+
```
Comment (kogonidze) [inline webpack.config.js:37]: Empty space

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                    },
+                    {
+                        loader: "react-svg-loader",
+                        options: {
+                            jsx: true
+                        }
+                    }
+                ]
+            }
+
```
Comment (kogonidze) [inline webpack.config.js:72]: Empty space

**Review by kogonidze — APPROVED** [review]
Well done

File: src/app/App.tsx (hunk 1 above)
Comment (Dmitry-Titov-K) [reply src/app/App.tsx]: fix

File: webpack.config.js (hunk 3 above)
Comment (Dmitry-Titov-K) [reply webpack.config.js]: fixed

File: webpack.config.js (hunk 4 above)
Comment (Dmitry-Titov-K) [reply webpack.config.js]: fixed

**Pull request merged**